examples/src/main/java/com/mapbox/navigation/examples/activity/TripServiceActivityKt.kt

```diff
@@ -83,8 +82,7 @@ class TripServiceActivityKt : AppCompatActivity(), OnMapReadyCallback {
                     }
                 }
 
-            mapboxTripService.startService()
-        }
+        mapboxTripService.startService()
```

Comment:
We should call `stopService` in `#onDestory`.

Reply:
👍